x/oracle/abci_test.go

```diff
@@ -127,9 +127,11 @@ var historacleTestCases = []struct {
 	expectedMaxOfHistoricMedians          sdk.Dec
 }{
 	{
-		[]string{"1.0", "1.2", "1.1", "1.4", "1.1", "1.15",
+		[]string{
+			"1.0", "1.2", "1.1", "1.4", "1.1", "1.15",
 			"1.2", "1.3", "1.2", "1.12", "1.2", "1.15",
-			"1.17", "1.1", "1.0", "1.16", "1.15", "1.12"},
+			"1.17", "1.1", "1.0", "1.16", "1.15", "1.12",
+		},
```

Comment:
why changing this?

Reply:
It's just a lint fix caused by my editor